dowhy/causal_refuters/placebo_treatment_refuter.py

```diff
@@ -22,6 +24,15 @@ class PlaceboTreatmentRefuter(CausalRefuter):
 
     :param random_state: The seed value to be added if we wish to repeat the same random behavior. If we want to repeat the same behavior we push the same seed in the psuedo-random generator.
     :type random_state: int, RandomState, optional
+
+    # using wording from joblib docs here since it's not that simple, depends on backend used
```

Comment:
Yeah no problem, I added them to the `CausalRefuter` class now. There isn't really a docstring there, so I left the parameter descriptions in the docstring of the child class (ie the specific refutation methods), is that okay or where should they go?

Also regarding the `backend` parameter, it says in the joblib docs we shoud use `prefer` and `require` instead if I understand correctly? Can you review that again as well please because maybe I misunderstand that. https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html

_"It is not recommended to hard-code the backend name in a call to Parallel in a library. Instead it is recommended to set soft hints..."_

Reply:
ah good catch! thanks for looking into it. 

In that case, I suggest that we do not specify the backend parameter at all, and lets users configure it using the joblib's context manager. It also makes our code simpler by not adding another parameter. By default, it will be 'loky' which will work for most users on a single machine. 

On causalrefuter: Oh, that skipped the earlier checks. That class should have a docstring :) Never mind, I can add it later in a future PR.